### PR TITLE
Misc TypeScript conversions

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
An atomization of #67809.

This PR aims to convert some miscellaneous UIs into TypeScript. Some of these are low hanging fruit, don't hate me, I am working up the self hatred to do more.

## Why It's Good For The Game
TypeScript is objectively better and provides great tools to debug code. We should really work to just convert all the UI into TSX 

## Changelog

:cl:
code: Refactored a large number of TGUI interfaces into TypeScript. If something's broken, report it!
/:cl:

### CODE REVIEW

**Comment by Hamcha** [comment]
As a heavy TSX user, this looks great!

Also I didn't know you could have initializers in your destructuring assignments!
```ts
const { proper_name, statuses = [], wires = [] } = data;
```

File: tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx [hunk 1]
```diff
@@ -2,19 +2,24 @@ import { useBackend } from '../backend';
...
+
   return (
     <Window width={350} height={300}>
       <Window.Content scrollable>
         <Section title="Alarms">
           <ul>
-            {priorityAlerts.length === 0 && (
-              <li className="color-good">No Priority Alerts</li>
+            {priority.length === 0 && (
+              <li className="color-good">No Priority </li>
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: Replace fail?

Comment (jlsnow301) [reply tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: They had just renamed the alerts in order to cast it to an array, this is unnecessary

Comment (Mothblocks) [reply tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: And what does that have to do with you removing "Alerts" from this text

Comment (jlsnow301) [reply tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: That is not what the ui data is called

Comment (Mothblocks) [reply tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: image

Where did this go jlsnow301

Comment (Hamcha) [reply tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: Mothblocks refers to the fact that you modified the user-visible text below as well (to "No priority " with a trailing space, too)

Comment (jlsnow301) [reply tgui/packages/tgui/interfaces/AtmosAlertConsole.tsx:20]: Oh oops! On it

**Pull request merged**